src/components/ArticlePage/Article.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FC } from "react";
+
+interface IProps {
+  date: string;
+  title: string;
+  subtitle: string;
+  paragraph: string;
+  source: string;
+  text: string;
+}
+export const Article: FC<IProps> = ({
+  date,
+  title,
+  subtitle,
+  paragraph,
+  source,
+  text
+}) => (
+  <div className="ml-6 mr-6 lg:ml-24">
+    <div className="lg:mr-24">
+      <div className="font-lato text-xs font-normal text-grey pt-4">{date}</div>
+      <div className="font-gilbert text-2xl font-bold text-black pb-4 pt-5">
```

Comment:
if this is title there should be h1 tag

Reply:
and there is also special article tag https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article